data-plane/pom.xml

```diff
@@ -397,7 +397,7 @@
             <!--  KinD e2e tests registry uses HTTP -->
             <allowInsecureRegistries>true</allowInsecureRegistries>
             <from>
-              <image>gcr.io/distroless/java17-debian11:nonroot</image>
+              <image>eclipse-temurin:17-jre-focal</image>
```

Comment:
Can we consider contributing to the distroless project to include s390x Java images?

There already have support for s390x in some images https://github.com/GoogleContainerTools/distroless/pull/612

Reply:
They have added arm64 recently https://github.com/GoogleContainerTools/distroless/pull/874 and we would just need to add s390x to the `ARCHITECTURES` array?
https://github.com/GoogleContainerTools/distroless/blob/b90f68384b5456e9ed6a37a1205d339fd431778b/java/BUILD#L10

Reply:
Well, there's a few things - on one hand, working with bazel on s390x is somewhat of a nightmare... the last bazel version I managed to compile and run on s390x is 3.7.2 which might be too old for GCT/distroless. On the other hand it's not sufficient to just add 's390x' to the ARCHITECTURES array. A whole bunch of additions would have to be made:
- debian.bzl
- checksums.bzl
- busybox_archives.bzl
- debian_versions.bzl
- debian_packages.yaml
- ...

And there's no guarantee that all the required Debian packages for s390x to support Java exist. Trying to add s390x support to GCT/distroless might open a whole can of worms... I'd prefer to switch the JRE for dispatcher and receiver to Eclipse Temurin Docker images seeing they are supporting all relevant architectures already (at least from IBM's perspective there's ppc64le support for kafka eventing, too which is not that far off).

What are the objections against Eclipse Temurin Docker images? Is it licensing concerns? Or the fact that they're hosted on Docker Hub (where pull rate limits apply)? Or that there are no rootless images available?